build_targets

```diff
@@ -1,6 +1,7 @@
 # crDroid build target list
 # <device> <build_type> <auto-upload> <upload-recovery>
 beryllium userdebug yes yes
+chef userdebug yes yes
```

Comment:
https://github.com/crdroidandroid/android_device_motorola_msm8998-common/blob/14.0/crdroid.dependencies#L5
remove default branch tag
14.0 is already the default, so no need to set it

https://github.com/crdroidandroid/android_device_motorola_msm8998-common/blob/14.0/crdroid.dependencies#L23
this is invalid
should host on crDroid Github/Gitlab

https://github.com/crdroidandroid/android_device_motorola_msm8998-common/blob/14.0/crdroid.dependencies#L28
same here

Reply:
Hi, i have changed repos for vendor
https://github.com/crdroidandroid/android_device_motorola_msm8998-common/commit/6f7541ffc867fd7f293bf4fce9030f1714a97a6e